tests/test_ml.py

```diff
@@ -35,6 +40,14 @@ def test_nearest_neighbor(self):
     def test_distance(self):
         self.assertAlmostEqual(distance((1, 2, 3), (1, 0, -1)), 4.47, 2)
 
+    def test_linear_regression(self):
+        param_0, param_1, rse, r_2 = simple_linear_regression(
+            self.data_set_linear_reg)
+
+        self.assertEqual(param_0, 1.2363636363636363)
+        self.assertEqual(param_1, 1.1696969696969697)
+        self.assertEqual(rse, 0.8384690232980003)
+        self.assertEqual(r_2, 0.0.952538038613988)
```

Comment:
Since we are using, try `assertAlmostEqual` in case there’s rounding errors (depending on platform/OS/python version)